ONSdigital/sdx-console
# Add Survey Switcher/Quantity

**Changes**
- Added a switcher to allow moving between different surveys, in order to ease having to lookup test data
- Added a quantity to allow surveys to be submitted multiple times

**How to test**
- Use as part of a compose setup and check surveys can be navigated between on main index page
- Increment the number of submissions and check it is transformed the desired number of times

**Who can test**
- Anyone but @iwootten
